clang/lib/DPCT/ASTTraversal.h

```diff
@@ -259,6 +259,26 @@ template <typename T> class NamedMigrationRule : public MigrationRule {
     emplaceTransformation(std::move(SIT));
   }
 
+  void analyzeUninitializedDeviceVar(const clang::Expr *Call,
+                                     const clang::Expr *Arg) {
+    if (!Call || !Arg)
+      return;
+    std::vector<const clang::VarDecl *> DeclsNeedToBeInitialized;
+    int Res = isArgumentInitialized(Arg, DeclsNeedToBeInitialized);
+    if (Res == 0) {
+      for (const auto D : DeclsNeedToBeInitialized) {
+        emplaceTransformation(new InsertText(
+            D->getEndLoc().getLocWithOffset(Lexer::MeasureTokenLength(
+                D->getEndLoc(), DpctGlobalInfo::getSourceManager(),
+                DpctGlobalInfo::getContext().getLangOpts())),
+            " = 0"));
```

Comment:
If there have the same issue with vector types, should we use zero initialization?

Reply:
vector types have default ctor to initialize their members.